src/sandbox/index.js

```diff
@@ -0,0 +1,13 @@
+const { SandboxClientBuilder } = require('./client.builder');
```

Comment:
Further to my previous comment, these would become for example: `const SandboxClient = require('./client.builder');`

Reply:
Fixed in 6c963d8